backend/app/store/formatter.go

```diff
@@ -48,7 +50,9 @@ func (f *CommentFormatter) FormatText(txt string) (res string) {
 		Flags: bf.Smartypants | bf.SmartypantsFractions | bf.SmartypantsDashes | bf.SmartypantsAngledQuotes,
 	})
 
-	res = string(bf.Run([]byte(txt), bf.WithExtensions(mdExt), bf.WithRenderer(rend)))
+	extRend := bfchroma.NewRenderer(bfchroma.Extend(rend), bfchroma.ChromaOptions(html.WithClasses()))
```

Comment:
Renderer has numerious [options](https://github.com/Depado/bfchroma#options), I wonder which set would work for us.

Reply:
From all I've read, current state should be enough and all the visual magic for white and dark themes should happen in CSS.